config/jobs/kubernetes-sigs/cluster-api/cluster-api-ci.yaml

```diff
@@ -0,0 +1,32 @@
+periodics:
+  kubernetes-sigs/cluster-api:
```

Comment:
periodics doesn't have a repo:jobs map (since they are not triggered from github repos), just a list of jobs here

Reply:
Got it. I've fixed this file and now all of the presubmit tests are passing.